stories/Who_was_protected_by_the_first_COVID-19_lockdown.ipynb

```diff
@@ -0,0 +1,2412 @@
+    "\n",
+    "There are two caveats that we have with the ONS analysis. The first is **reproducibility**. We want to confirm the ONS results by making analysis procedure open. The second caveat is that the ONS report **aggregates data over time**, and therefore that it might miss interesting differences in outcomes between the different stages of lockdown. Between March and May represents the time when the lockdown was most severe, with measures relaxing from June onwards. We wonder whether the ONS analysis will continue to be relevant as lockdown eases. For this purpose, we wish to extend the ONS analysis to cover all available data, and at the same time, make a comparison between the different stages of lockdown.\n",
+    "\n",
+    "Thus for our first story we ask:\n",
+    "\n",
+    "**Have the COVID-19 lockdown measures protected people equally across all socio-economic groups in society?**\n",
+    "\n",
+    "We have two main objectives\n",
+    "\n",
+    "1) We want to replicate the ONS analysis using their provided data to ensure that we have all the inputs necessary to understand the problem.\n",
+    "\n",
+    "2) We want to extend the ONS analysis to consider different time periods - representing the severity of the different stages of lockdown - to see how this affects people from different socio-economic groups.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Key Metrics\n",
+    "\n",
+    "Our analysis will involve exploring the relationship between the following key metrics:\n",
+    "\n",
+    "- COVID-19 mortality rates over time and across geographical regions.\n",
+    "- Index of multiple deprivation (IMD) by geographical region - a measure of the geographic spread of social deprivation (see [definition](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/835115/IoD2019_Statistical_Release.pdf) and [explanation](https://en.wikipedia.org/wiki/Multiple_deprivation_index)).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Data Sources\n",
+    "\n",
+    "We will use the following datasets:\n",
+    "\n",
+    "- [Mortality count time series](https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/causesofdeath/datasets/deathregistrationsandoccurrencesbylocalauthorityandhealthboard)\n",
+    "- [IMD Rankings (England only)](https://www.gov.uk/government/statistics/english-indices-of-deprivation-2019)\n",
+    "- [Populations](https://www.ons.gov.uk/file?uri=%2fpeoplepopulationandcommunity%2fpopulationandmigration%2fpopulationestimates%2fdatasets%2fpopulationestimatesforukenglandandwalesscotlandandnorthernireland%2fmid20182019laboundaries/ukmidyearestimates20182019ladcodes.xls)\n",
```

Comment:
This link and a couple of others open the file directly - this is really good for ease of use, but it doesn't illustrate where the file was obtained from. Could you add a note to clarify the source? Something like:
`- [Populations](<link to file here>), provided by [ONS](<link to ONS page where file is described>)`